### PR TITLE
Persist tooltips on hover

### LINKED ISSUE #23958: Accessibility regression: Tooltips don't persist on hover any longer
**Describe the bug**
Tooltips should persist on hover.

This feature was previously implemented in https://github.com/WordPress/gutenberg/issues/7004 / https://github.com/WordPress/gutenberg/pull/8033

However, in the Gutenberg version currently shipped in WordPress 5.5 Beta 2, tooltips don't persist on hover any longer.

I made two animated GIF to better illustrate:

WordPress 5.4:

tooltips persist

WordPress 5.5 Beta 2:

tooltips do not persist

There are 3 requirements for accessible tooltips. For more details, see https://github.com/WordPress/gutenberg/pull/8033

Reference:

**WCAG Success Criterion 1.4.13 Content on Hover or Focus**
https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus
https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

- dismissable
- hoverable
- persistent

Worth noting that #8033 made the tooltips persist but the "dismissable" part is missing. This part is still waiting for a solution,  see https://github.com/WordPress/gutenberg/issues/15145

In the meantime, the lack of persistence on hover is a clear regression "in trunk" and should be remediated before WordPress 5.5 is released.

### PR DESCRIPTION
**Pull request by ntsekouras**
## Description
Fixes: https://github.com/WordPress/gutenberg/issues/23958

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/components/src/tooltip/style.scss [hunk 1]
```diff
@@ -17,9 +17,6 @@
 	font-size: 12px;
 	box-shadow: none;
 
-	// This prevents quickly hovering the tooltip from blocking hovering something else.
-	pointer-events: none;
```
Comment (youknowriad) [inline packages/components/src/tooltip/style.scss:21]: @jasmussen do you remember why we add this? (It was added on the original G2 branch).

**Comment by ItsJonQ** [comment]
Tested locally, and it looks like it resolves the issue! Tooltips are able to persist.

It looks like it was added during the G2 design refresh. I feel like there's a reason for this, but I'm lacking the context.
I know the Toolbar was iterated on a bunch during that phase. Perhaps this change was once needed, but not so much anymore?

Dropdowns, Top Toolbar mode, and typical mouse-based navigation still work.

I'm comfortable with this fix @ntsekouras !

🚀

File: packages/components/src/tooltip/style.scss (hunk 1 above)
Comment (MichaelArestad) [reply packages/components/src/tooltip/style.scss:21]: From the comment, it looks like it was added to make the tooltip "invisible" to a cursor so a user wouldn't accidentally click it when trying to click something else in the UI or on the canvas.

**Review by youknowriad — APPROVED** [review]
I tested this as well and it seems to work great, I can't remember why this has been added.

**Comment by afercia** [comment]
Thanks for fixing this so quickly. This should be included in WP 5.5 though, what is the best way to make sure that happens? @youknowriad

**Comment by youknowriad** [comment]
yes, It has the "backport to Core" label, it means it will be backported before the next WordPress 5.5 beta.

File: packages/components/src/tooltip/style.scss (hunk 1 above)
Comment (jasmussen) [reply packages/components/src/tooltip/style.scss:21]: I think Michael is right, yes. If you hover a button and see a tooltip, that might cover a button immediately above it, and if you quickly click there to click that button, you should be able to.

**Pull request merged**